demo/README.md

```diff
@@ -0,0 +1,8 @@
+# simple example
```

Comment:
We should not use the word "simple". As it may be simple to us, but not for everyone else :)
So could you please just write "Example"?